### PR TITLE
Replace release notes page with github link to avalanchego releases

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
This PR removes redundant "Release Notes" documentation from the dev docs. The AvalancheGo documentation now contains a link in the sidebar to the release notes in GitHub for reference.

Broken links were referenced the `_redirects` file to prevent navigation to broken links from external sources. 
References to the pre existing release notes pages were replaced with hardcoded links to Github. 

Please visit the dev link in the Cloudflare workflow to view the staged changes.
 
Current State: https://docs.avax.network/apis/avalanchego/avalanchego-release-notes

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -369,3 +369,11 @@
 /subnets/README /learn/avalanche/subnets-overview 301
 /overview/projects /learn/projects 301
 /subnets /learn/avalanche/subnets-overview 301
+
+#05/24/2023
+/apis/avalanchego/avalanchego-release-notes https://github.com/ava-labs/avalanchego/releases 301
+/apis/avalanchego/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
+/apis/avalanchego/avalanchego-release-notes.md https://github.com/ava-labs/avalanchego/releases 301
```
Comment (yulin-dong) [inline static/_redirects:376]: This is not needed as currently https://docs.avax.network/apis/avalanchego/avalanchego-release-notes.md is not a valid page.

File: static/_redirects [hunk 2]
```diff
@@ -369,3 +369,11 @@
...
+/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
+/avalanchego-release-notes.md#v1912-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.9.12 301
```
Comment (yulin-dong) [inline static/_redirects:378]:
```suggestion
/avalanchego-release-notes#v1912-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.9.12 301
```

File: static/_redirects [hunk 3]
```diff
@@ -369,3 +369,11 @@
 /subnets/README /learn/avalanche/subnets-overview 301
 /overview/projects /learn/projects 301
 /subnets /learn/avalanche/subnets-overview 301
+
+#05/24/2023
+/apis/avalanchego/avalanchego-release-notes https://github.com/ava-labs/avalanchego/releases 301
+/apis/avalanchego/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
+/apis/avalanchego/avalanchego-release-notes.md https://github.com/ava-labs/avalanchego/releases 301
+/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
```
Comment (yulin-dong) [inline static/_redirects:377]: What is the reason for this and following two URLs?

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: static/_redirects [hunk 4]
```diff
@@ -369,3 +369,11 @@
 /subnets/README /learn/avalanche/subnets-overview 301
 /overview/projects /learn/projects 301
 /subnets /learn/avalanche/subnets-overview 301
+
+#05/24/2023
+/apis/avalanchego/avalanchego-release-notes https://github.com/ava-labs/avalanchego/releases 301
+/apis/avalanchego/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
```
Comment (yulin-dong) [inline static/_redirects:375]: This is no longer needed.

File: static/_redirects (hunk 1 above)
Comment (meaghanfitzgerald) [reply static/_redirects:376]: If external sites are referencing https://docs.avax.network/apis/avalanchego/avalanchego-release-notes we'd want to redirect them to the official notes in GH

File: static/_redirects (hunk 3 above)
Comment (meaghanfitzgerald) [reply static/_redirects:377]: If external sites are referencing https://docs.avax.network/apis/avalanchego/avalanchego-release-notes we'd want to redirect them to the official notes in GH

**Comment by yulin-dong** [comment]
Where is the preview page for this PR?

File: static/_redirects (hunk 3 above)
Comment (yulin-dong) [reply static/_redirects:377]: > If external sites are referencing https://docs.avax.network/apis/avalanchego/avalanchego-release-notes we'd want to redirect them to the official notes in GH

But the first part is /avalanchego-release-notes/, not /apis/avalanchego/avalanchego-release-notes

**Comment by meaghanfitzgerald** [comment]
@yulin-dong https://github.com/ava-labs/avalanche-docs/pull/1368/checks?check_run_id=13737695951

File: static/_redirects [hunk 5]
```diff
@@ -369,3 +369,8 @@
 /subnets/README /learn/avalanche/subnets-overview 301
 /overview/projects /learn/projects 301
 /subnets /learn/avalanche/subnets-overview 301
+
+#05/24/2023
+/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301
+/avalanchego-release-notes#v1912-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.9.12 301
+/avalanchego-release-notes#v1714-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.7.14 301
```
Comment (yulin-dong) [inline static/_redirects:376]: Please manually verify the first part is a valid page on https://docs.avax.network/. These two are not valid.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: static/_redirects (hunk 5 above)
Comment (meaghanfitzgerald) [reply static/_redirects:376]: https://docs.avax.network/apis/avalanchego/avalanchego-release-notes is the currently valid page that is being removed, which is why I originally had this line to redirect:
`/apis/avalanchego/avalanchego-release-notes/ https://github.com/ava-labs/avalanchego/releases 301`

I am a bit confused

File: sidebars.js [hunk 6]
```diff
@@ -291,11 +295,6 @@ const sidebars = {
   community: [
     "community/README",
     "community/bug-bounty",
-    {
```
Comment (Andyvargtz) [inline sidebars.js:294]: Why was this removed?

File: static/_redirects (hunk 5 above)
Comment (yulin-dong) [reply static/_redirects:376]: I removed /apis/avalanchego/avalanchego-release-notes/, but there is still /apis/avalanchego/avalanchego-release-notes which should be kept.

Comment (meaghanfitzgerald) [reply static/_redirects]: Understood. Verified each link manually and corrected the redirects. Thank you for the explanation/patience.

**Review by yulin-dong — APPROVED** [review]
